src/components/token-info/TokenInfoItem.js

```diff
@@ -86,23 +89,50 @@ export default function TokenInfoItem({
         {asset ? (
           <TokenInfoBalanceValue align={align} asset={asset} isNft={isNft} />
         ) : (
-          <ButtonPressAnimation
-            enableHapticFeedback={!!onPress && enableHapticFeedback}
-            onPress={onPress}
-            scaleTo={1}
-          >
-            <TokenInfoValue
-              activeOpacity={0}
-              align={align}
-              color={color}
-              isNft={isNft}
-              lineHeight={lineHeight}
-              size={size}
-              weight={weight}
-            >
-              {!loading && children}
-            </TokenInfoValue>
-          </ButtonPressAnimation>
+          <>
+            <Columns alignHorizontal="left">
+              <Column alignHorizontal="left">
+                <ButtonPressAnimation
+                  enableHapticFeedback={!!onPress && enableHapticFeedback}
+                  onPress={onPress}
+                  scaleTo={1}
+                >
+                  <TokenInfoValue
+                    activeOpacity={0}
+                    align={align}
+                    color={color}
+                    isNft={isNft}
+                    lineHeight={lineHeight}
+                    size={size}
+                    weight={weight}
+                  >
+                    {!loading && children}
+                  </TokenInfoValue>
+                </ButtonPressAnimation>
+              </Column>
+              {showEditButton && (
```

Comment:
Looks good. My only worry is that this might not be as composable if we end up choosing to have an addon component or such for `TokenInfoItem`. 

Another approach could be that we render a prop called `addonComponent` here, and then in `ENSBriefTokenInfoRow` we do something like:

```
  addonComponent={
    showEditButton && (
      <Column>
        ...
      </Column>
  )}
```

Reply:
done!